config/routes.rb

```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
-  root 'articles#index'
+  root 'sessions#login'
 
-  resources :articles
+  get  '/login',   to: 'sessions#login'
+  post '/login',   to: 'sessions#create'
+  get  '/logout',  to: 'sessions#logout'
+
+  resources :users, only: [:new, :create]
```

Comment:
@JuanMaGiraldo how the resources helper method works?

Reply:
This method allows to declare the most common routes to carry out CRUD operations. In this case, I used the only operator to create the specified routes, so, it will not create the routes for: index, show, edit, update and destroy.